relay-config/src/config.rs

```diff
@@ -736,6 +736,21 @@ impl Default for Http {
     }
 }
 
+fn default_persistent_buffer_path() -> PathBuf {
+    PathBuf::from("envelope-buffer.db")
+}
+
+/// Controls internal caching behavior.
+#[derive(Serialize, Deserialize, Debug)]
+struct PersistentBuffer {
+    /// True if the Relay should persist the incoming envelopes to the disk in case they cannot be
+    /// processed right away.
+    enabled: bool,
```

Comment:
IMO this flag is redundant, because to disable persistence we can just comment out `persistent_envelope_buffer`.